com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/archive/ManifestBuilder.java

```diff
@@ -325,6 +108,22 @@ public ManifestBuilder(boolean outputManifestHash) {
         this.outputManifestHash = outputManifestHash;
     }
 
+    public static ManifestBuilder create() {
+        return create(false);
+    }
+
+    public static ManifestBuilder create(boolean outputManifestHash) {
```

Comment:
Helper to create a ManifestBuilder instance with the correct cryptographic algorithms for resource hash calculations and signatures. We need to make sure we can get the algorithm in different places.